src/image_processing/process_node.cpp

```diff
@@ -0,0 +1,55 @@
+#include <cv_bridge/cv_bridge.h>
```

Comment:
Add comments in this file. I also suggest u look at the other code in motion folder and see how to pass the topic names from the launch file.